airgun/entities/computeresource.py

```diff
@@ -0,0 +1,73 @@
+from navmazing import NavigateToSibling
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.computeresource import (
+    ComputeResourcesView,
+    ResourceProviderDetailsView,
+)
+
+
+class ComputeResourceEntity(BaseEntity):
+
+    def create(self, values):
+        view = self.navigate_to(self, 'New')
+        view.fill(values)
+        view.submit.click()
+
+    def search(self, value):
+        view = self.navigate_to(self, 'All')
+        return view.search(value)
+
+    def edit(self, name, values):
+        view = self.navigate_to(self, 'Edit', entity_name=name)
+        view.fill(values)
+        view.submit.click()
+
+    def read(self, entity_name):
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        return view.read()
+
+    def delete(self, entity_name):
+        view = self.navigate_to(self, 'Delete', entity_name=entity_name)
```

Comment:
I wouldn't say that delete is a separate navigation point, so in my opinion it should be like this:
```
view = self.navigate_to(self, 'All')
view.search(value)
view.action_list.fill('Delete')
```